src/client/Client.cc

```diff
@@ -607,6 +607,10 @@ void Client::shutdown()
     std::lock_guard l{client_lock};
     ceph_assert(initialized);
     initialized = false;
+  }
+
+  {
+    std::lock_guard lock(timer_lock);
```

Comment:
```suggestion
    std::scoped_lock lock(timer_lock);
```

is the better interface now. Please use that instead.

Reply:
Done.